tests/Umbraco.Tests.AcceptanceTest/tests/ApiTesting/Dictionary/Dictionary.spec.ts

```diff
@@ -0,0 +1,83 @@
+import {test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
+test.describe('Dictionary tests', () => {
+  const dictionaryName = 'Word'
+
+  test.beforeEach(async ({page, umbracoApi}) => {
+    await umbracoApi.dictionary.ensureDictionaryNameNotExists(dictionaryName);
+  });
+
+  test.afterEach(async ({page, umbracoApi}) => {
+    await umbracoApi.dictionary.ensureDictionaryNameNotExists(dictionaryName);
+  })
+
+  test('can create a dictionary', async ({page, umbracoApi, umbracoUi}) => {
+    const translationData = [
+      {
+        'isoCode': 'en-US',
+        'translation': 'Word'
+      },
+      {
+        'isoCode': 'da-DK',
+        'translation': 'Ord'
+      }
+    ];
+
+    await umbracoApi.dictionary.createDictionary(dictionaryName, translationData);
+
+    // Assert
+    await expect(umbracoApi.dictionary.doesDictionaryWithNameExists(dictionaryName)).toBeTruthy();
+  });
+
+  test('can update a dictionary', async ({page, umbracoApi, umbracoUi}) => {
+    const oldDictionaryName = 'OldWord';
+
+    await umbracoApi.dictionary.createDictionary(oldDictionaryName);
+
+    const dictionary = await umbracoApi.dictionary.getDictionaryByName(oldDictionaryName);
+
+    // Updates the dictionary name
+    dictionary.name = dictionaryName;
+    await umbracoApi.dictionary.updateDictionaryById(dictionary.id, dictionary);
```

Comment:
I think the id should be a given here, the `dictionary` object should have it, no need to pass it in